conf.d/pure.fish

```diff
@@ -1,4 +1,4 @@
-set --universal pure_version 2.1.5 # used for bug report
+set --universal pure_version 2.1.6 # used for bug report
```

Comment:
check latest release for correctness

Reply:
You're actually the one that bumped it. I can make changes if necessary.

Reply:
That's a note for the future, I merge a PR yesterday and release a 2.1.6 version so this need to be updated accordingly.

I haven't had time to finish this PR, sorry.

Reply:
Oh no problem at all. I thought you were telling me to do it.